Documentation/Metrics/arangodb_internal_cluster_info_memory.yaml

```
@@ -0,0 +1,18 @@
+name: arangodb_internal_cluster_info_memory
```

I suggest to rename this metric to "_memory_usage", as it then would be in line with several other metrics.
```suggestion
name: arangodb_internal_cluster_info_memory_usage
```